code/modules/client/widescreen.dm

```diff
@@ -0,0 +1,135 @@
+/client/proc/change_view(new_size)
+	if (isnull(new_size))
+		CRASH("change_view called without argument.")
+
+	view = new_size
+	mob.reload_fullscreen()
+	if (isliving(mob))
+		var/mob/living/M = mob
+		M.update_damage_hud()
+	addtimer(CALLBACK(src,.verb/fit_viewport,10)) //Delayed to avoid wingets from Login calls.
```

Comment:
Wouldn't it be easier to just call this outside of the Login proc?

I'm not sure what a winget is, as I'm not familiar with Byond internals.

Reply:
That's a comment from TG, I am not too sure either, pretty sure it's to call the fit viewport verb once the windows loaded.